bin/compile

```diff
@@ -236,7 +236,7 @@ export PATH=/app/bin:${VENDORED_NGINX}/sbin:${VENDORED_PHP}/sbin:${VENDORED_PHP}
 export APP_BUILD_TIME=$(date +%Y%m%d%H%M%S)
 SH
 
-if [ $STACK = "scalingo" ] ; then
+if [ $STACK = "scalingo-14" ] ; then
```

Comment:
Not needed on scalingo-18 ?

Reply:
Nope, tail is recent enough on ubuntu 18.04